src/wp-admin/css/common.css

```
@@ -1289,12 +1296,14 @@ th.action-links {
 }
 
 .filtered-by .tags {
-	display: inline;
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: wrap;
+        gap: 3px;
 }
 
 .filtered-by .tag {
-	margin: 0 5px;
-	padding: 4px 8px;
+	padding: 3px 5px;
```

```suggestion
	padding: 4px 8px;
```

I prefer the padding as it was.